src/pages/account/mijn-mappen/[collectionSlug]/index.tsx

```diff
@@ -150,9 +150,33 @@ const AccountMyCollections: NextPage = () => {
 	};
 
 	const onRemoveFromCollection = (item: IdentifiableMediaCard, collection: Collection) => {
-		collectionsService.removeFromCollection(collection.id, item.schemaIdentifier).then(() => {
-			collectionMedia.refetch();
-		});
+		collectionsService
+			.removeFromCollection(collection.id, item.schemaIdentifier)
+			.then((response) => {
+				if (response === undefined) {
+					return;
+				}
+
+				collectionMedia.refetch();
+
+				const descriptionVariables = {
+					item: item.name,
+					collection:
+						collection?.name ||
+						t('pages/account/mijn-mappen/collection-slug/index___onbekend'),
+				};
+
+				toastService.notify({
+					maxLines: 3,
+					title: t(
+						'pages/account/mijn-mappen/collection-slug/index___item-verwijderd-uit-map'
+					),
+					description: t(
+						'pages/account/mijn-mappen/collection-slug/index___item-is-verwijderd-uit-collection',
+						descriptionVariables
+					),
+				});
+			});
```

Comment:
Better error handling
Prefer async await to avoid unhandled promise rejections

```suggestion
		try {
			  const response = await collectionsService
				  .removeFromCollection(collection.id, item.schemaIdentifier)
			    if (response === undefined) {
				    throw new Error('response was empty');
			    }
		    
			    collectionMedia.refetch();
		    
			    const descriptionVariables = {
				    item: item.name,
				    collection:
					    collection?.name ||
					    t('pages/account/mijn-mappen/collection-slug/index___onbekend'),
			    };
		    
			    toastService.notify({
				    maxLines: 3,
				    title: t(
					    'pages/account/mijn-mappen/collection-slug/index___item-verwijderd-uit-map'
				    ),
				    description: t(
					    'pages/account/mijn-mappen/collection-slug/index___item-is-verwijderd-uit-collection',
					    descriptionVariables
				    ),
			    });
			} catch (err) {
				toastService.notify({
					title: t(
						'Error'
					),
					description: t(
						'Het verwijderen van het item uit de map is mislukt
					),
				});
			}
```